### ISSUE TITLE
Finetune Presto with masking of specific input channels

### ISSUE THREAD

**Issue by kvantricht** [WorldCereal/presto-worldcereal#92]
Because training data largely comes from the situation where Sentinel-1B was still up and running, we now finetune with Sentinel-1 always present, while in inference it will not always be the case. We should adapt the finetuning strategy to be able to mask some channels also during finetuning.

**Comment by kvantricht** [comment]
Closed by https://github.com/WorldCereal/presto-worldcereal/pull/102
